sound/soc/sof/core.c

```diff
@@ -214,6 +218,7 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 	} else {
 		dev_dbg(sdev->dev, "SOF firmware trace disabled\n");
 	}
+#endif
```

Comment:
this whole logic looks odd. We've want the ability to override Kconfig choices with the module parameter, that's why I suggested in https://github.com/thesofproject/linux/pull/3021/commits/2513c70df497dc2ed5aa3977166a5e0a2b7efb48

Reply:
This is still the optional code and it is going to be removed later.

Reply:
this needs to be rebased, this was changed in topic/sof-dev.